internal/auto_download.py

```diff
@@ -11,7 +11,7 @@
 
 def download_latest():
     """main download function"""
-    seven_zip_paths = ["7z.exe", "C:/Program Files/7-Zip/7z.exe"]
+    seven_zip_paths = ["7z.exe", "C:/Program Files/7-Zip/7z.exe"] if os.name == 'nt' else ['7z']
```

Comment:
Basicly 7z is not neccessary on anything beside windows, on OSX its a dmg file which needs to be mounted and then i think something needs to get copied around and unter linux its unzip or something like tar if simc would be packed as a *.tar.gz. Like mentioned at the other place linux isn't available and osx i'm unsure how to try it :)

Reply:
Yeah additionally the download only looks at the latest win64 file too, i was being dumb thinking this was all that was involved.